### PR TITLE
Fixed that miss colored '1' button on the safe and nuke keypad

### PR DESCRIPTION
**Pull request by WarlockD**
## About The Pull Request

Its been driving me crazy.  So I fixed it.  I also went a head and set the wall/captin safe to the "retro" template like the nuke button is so all the shadows come in right.  While I was at it I removed the obsoleted Grid control and just did it all with Flex.
## Why It's Good For The Game

The keypad is more consistent.
2pxgJs0id3 | nBIAozJxok
## Changelog
:cl:
fix: The '1' button is the right color
fix: Backgrounds look better on normal safes
:cl:

### CODE REVIEW

File: tgui/packages/tgui/styles/interfaces/NuclearBomb.scss [hunk 1]
```diff
@@ -32,12 +32,6 @@ $background-beige: #e8e4c9;
   }
 }
 
-.NuclearBomb__Button--1 {
-  background-color: #d3cfb7 !important;
-  border-color: #d3cfb7 !important;
-  color: #a9a692 !important;
-}
```
Comment (MrMelbert) [inline tgui/packages/tgui/styles/interfaces/NuclearBomb.scss:39]: It's intentionally discolored, that's why it's a different class

Comment (MrMelbert) [reply tgui/packages/tgui/styles/interfaces/NuclearBomb.scss:39]: image

**Comment by JohnFulpWillard** [comment]
This was all intentional so you need to justify why we should change the intended look, not that is a fix

**Comment by DingleFlop** [comment]
Perhaps adding a hover effect, instead of having a single, statically discolored button, would be a reasonable compromise. The best of both worlds, even.

File: tgui/packages/tgui/styles/interfaces/NuclearBomb.scss (hunk 1 above)
Comment (WarlockD) [reply tgui/packages/tgui/styles/interfaces/NuclearBomb.scss:39]: Humm see I was wondering about why its been like that for so long then or why it had its own class. I will set it up so you can have the main device that uses the pad send classes for it instead.  Humm  kind of makes me wonder if I can put random "smudges" or fingerprints on some of the keys.  Let me play with it.

Comment (WarlockD) [reply tgui/packages/tgui/styles/interfaces/NuclearBomb.scss]: i5hpreOPp6
There we go, the normal safe keypad is on a separate css class.

**Comment by WarlockD** [comment]
Ok did some minor updates.  Just changed the NukeBomb to be Typescript, moved it to Flex, then set each on their own class.  With 516 on the way I didn't want to use ms-grid to fix all the alignment issuse.

**Pull request closed without merging**